src/flynt/api.py

```diff
@@ -182,10 +187,14 @@ def fstringify(
         return 0
 
 
-def _resolve_files(files_or_paths) -> List[str]:
+def _resolve_files(files_or_paths, excluded_files_or_paths=None) -> List[str]:
```

Comment:
the only call to the `_resolve_files` has a value for `excluded_files_or_paths`. Let's remove the default value - it's not used and I see no design benefit in it.

Reply:
done !